### PR TITLE
docs(tabs): updates Tabs component docs to match other components

### PR DESCRIPTION
**Pull request by anechol**
# Description

Updates Tabs documentation to match other components.

[DSS-1192](https://kajabi.atlassian.net/browse/DSS-1192)

## Type of change

- [x] Documentation (updates component documentation)

[DSS-1192]: https://kajabi.atlassian.net/browse/DSS-1192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/src/components/pds-tabs/docs/pds-tabs.mdx [hunk 1]
```diff
@@ -3,62 +3,64 @@ import { components } from '../../../../dist/docs.json';
...
-  <pds-tabpanel name="Waffle">Content Waffle</pds-tabpanel>
-</pds-tabs>
+  <pds-tabs active-tab-name="Dollop" variant="primary" component-id="primary" tablist-label="Foo">
+    <pds-tab name="Sturdy">Sturdy</pds-tab>
+    <pds-tab name="Dollop">Dollop</pds-tab>
+    <pds-tab name="Waffle">Waffle</pds-tab>
+    <pds-tabpanel name="Sturdy">Content Sturdy</pds-tabpanel>
+    <pds-tabpanel name="Dollop">Content Dollop</pds-tabpanel>
+    <pds-tabpanel name="Waffle">Content Waffle</pds-tabpanel>
+  </pds-tabs>
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-tabs/docs/pds-tabs.mdx]: I'll fix this, but I wanted to bring it to your attention. When making this change it adds space in the code example. This occurs due to how backtick works.

File: libs/core/src/components/pds-tabs/docs/pds-tabs.mdx [hunk 2]
```diff
@@ -3,41 +3,43 @@ import { components } from '../../../../dist/docs.json';
...
 # Tabs
 
-A tabs component to generate all versions of `pds-tab` as outlined in [Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%F0%9F%A7%A9-Sage-components?node-id=488%3A43471&t=5FtThY98rugq7hGi-1).
+The Tabs component is used to separate and display different panels of related content activated by tab buttons.
 
-## Sub Components
+## Guidelines
 
-### Tab
+### Accordions vs Tabs
```
Comment (pixelflips) [inline libs/core/src/components/pds-tabs/docs/pds-tabs.mdx:10]: While important, should this be after when and when not to use?

**Review by pixelflips — APPROVED** [review]
I small non-blocking comment, otherwise LGTM!

**Review by ju-Skinner — APPROVED** [review]

File: libs/core/src/components/pds-tabs/docs/pds-tabs.mdx (hunk 1 above)
Comment (anechol) [reply libs/core/src/components/pds-tabs/docs/pds-tabs.mdx]: Ahhh thanks for catching that. Didn't think that would have an effect.

File: libs/core/src/components/pds-tabs/docs/pds-tabs.mdx (hunk 2 above)
Comment (anechol) [reply libs/core/src/components/pds-tabs/docs/pds-tabs.mdx:10]: We don't really have an established pattern for this, but I can see the reasoning for moving it. It's been updated along with the Accordion component.

**Pull request merged**